runtime/drivers/duckdb/duckdb.go

```diff
@@ -145,7 +146,7 @@ func (d Driver) Open(instanceID string, cfgMap map[string]any, ac *activity.Clie
 		logger.Warn("failed to install embedded DuckDB extensions, let DuckDB download them", zap.Error(err))
 	}
 
-	cfg, err := newConfig(cfgMap)
+	cfg, err := newConfig(cfgMap, st.DataDir("duckdb"))
```

Comment:
See above comment – then you can avoids, which has the problem of not supporting multiple connectors that use the same driver.